### DISCUSSION TITLE
Deep neural  network Deployment on ZCU104 board

### DISCUSSION THREAD

**Discussion by ucanakwenze** [beetleboxorg/BeetleboxCI-Community#2] [Q&A]
Good day,
Please I am A PhD student at De Montfort University UK. I am trying to deploy a deep neural network on the zcu104 Xilinx board. I have seen your work on COVID with zcu104. I have installed Virtualbox and Ubuntu on my Windows laptops. I have installed Vitis AI docker image in the Linux. Also, I have used the renet50 in the model zoo and followed the 2.5 user guide,  I did quantization and compiled the model but was unable to deploy the model. I need support because I have spent so much time on this.  I will pay for the support. you can get me at ucanakwenze@gmail.com

Thanks

Uche

**Comment by AndrewSwirski** [comment]
Hi Uche,

I wouldn't recommend leaving your personal email on a public forum where anybody can view it. 

BeetleboxCI does support Vitis AI and you don't need to pay us. Just have one of your supervisors apply for our university programme and we would be happy to install BeetleboxCI at De Montfort. We could then help you with building automated pipelines for Vitis AI.

You can book a meeting here: https://userportal.beetleboxci.com/signup

Regards

Andrew
